base/applications/notepad/main.h

```diff
@@ -41,12 +41,11 @@ typedef enum
     ENCODING_UTF16LE =  1,
     ENCODING_UTF16BE =  2,
     ENCODING_UTF8    =  3,
-    ENCODING_UTF8BOM =  4
+    ENCODING_UTF8BOM =  4,
+    /* ENCODING_DEFAULT = ENCODING_ANSI, */
+    ENCODING_DEFAULT = ENCODING_UTF8,
 } ENCODING;
 
```

Comment:
Hm... Move "ENCODING_DEFAULT" as a #define instead ?

```suggestion
    ENCODING_UTF8BOM =  4
} ENCODING;

#define ENCODING_DEFAULT    ENCODING_UTF8 // ENCODING_ANSI

```